src/cloud-code/Parse.Cloud.js

```diff
@@ -528,6 +529,37 @@ ParseCloud.beforeConnect = function (handler, validationHandler) {
   );
 };
 
+/**
+ * Sends an email through the Parse Server mail adapter.
+ *
+ * **Available in Cloud Code only.**
+ * **Requires a mail adapter to be configured for Parse Server.**
+ *
+ * ```
+ * Parse.Cloud.sendMail({
+ *   from: 'Example <test@example.com>',
+ *   to: 'contact@example.com',
+ *   subject: 'Test email.',
+ *   text: 'This email is a test'
+ * });
+ *```
+ *
+ * @method sendMail
+ * @name Parse.Cloud.sendMail
+ * @param {Object} data The object of the mail data to send
+ */
+ParseCloud.sendMail = function (data) {
```

Comment:
See [this comment](https://github.com/parse-community/docs/pull/806#discussion_r548999827); please change to `sendEmail`.